common/test/test_msgpack_serializer.py

```diff
@@ -0,0 +1,87 @@
+from common.serializers.msgpack_serializer import MsgPackSerializer
+
+serializer = MsgPackSerializer()
+
+
+def check(value):
+    assert value == serializer.deserialize(serializer.serialize(value))
+
+
+def test_serialize_int():
+    check(1)
+    check(111324324324324)
+    check(-1)
+    check(-44445423453245)
+    check(0)
+
+
+def test_serialize_str():
+    check('aaaa')
+    check('abc def')
+    check('a b c d e f')
+    check('строка')
+
+
+def test_serialize_list():
+    check([1, 'a', 'b'])
```

Comment:
Unfortunately, msgpack can work with either tuples or lists (lists by default).
Actually, in a real code we almost always pass dict as an input.